english/onlyawoman.md

```diff
@@ -0,0 +1 @@
+Only a woman could fill that aching hole in his heart that a woman tore out long ago.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.